bin/console-conf-wrapper

```diff
@@ -80,6 +81,7 @@ if [ "$(snap managed)" = "true" ]; then
         done
     else
         touch /var/lib/console-conf/complete
```

Comment:
Does this file still matter?

Reply:
Yes for 2 reasons.

The next time we start this service, we will check for this file without having to go through all the checks. And that keeps it consistent with previous behavior, that is if a password was there, but later removed, we should still show the login screen.

The second reason is that snapd does look at this file.